src/components/search-result-list/SearchResultList.tsx

```diff
@@ -24,15 +37,24 @@ const SearchResultList: React.FC<SearchResultListProps> = ({ resultItems }) => {
           </li>
         ))}
       {worksAreLoaded &&
-        resultItems.map((item, i) => (
-          <li key={item.workId}>
-            <SearchResultListItem
-              item={item}
-              coverTint={getCoverTint(i)}
-              resultNumber={i + 1}
-            />
-          </li>
-        ))}
+        resultItems.map((item, i) => {
+          const isFirstNewItem = i === page * pageSize;
+          return (
+            <li
+              key={item.workId}
+              ref={isFirstNewItem ? lastItemRef : null}
+              // Because we're using a ref to focus the first item in the new page when pagination occurs.
+              // we need to remove focus ( set tabIndex -1), so that it can be set programmatically.
+              tabIndex={-1}
```

Comment:
```suggestion
              // We use a ref to focus the first item in the new page programmatically when pagination occurs.
              // Set tabIndex -1 to support this without allowing keyboard focus. We have just as appropriate 
              // elements within the item suitable for keyboard focus.
```

I tried to reword this a bit. Use it if it makes sense to you.